src/diceBot/SwordWorld2_0.rb

```diff
@@ -142,8 +142,9 @@ def is2dCritical
   end
 
   # SW2.0 の超成功用
+  # @param (see DiceBot#check2dCritical)
```

Comment:
処理には直接影響しませんが、YARDが `@param` の書式がおかしいと警告を出すので…（`{ }` はリンクのための書式です）

```suggestion
  # SW2.0 の超成功用
  # 
  # 引数については {DiceBot#check2dCritical} を参照。
```

Reply:
yardのリファレンス通りに似たようなことを書きたいのですがどうしたらいいんだろう
https://www.rubydoc.info/gems/yard/file/docs/Tags.md#reference-tags

```
# @param (see #clean)
def activate(user, host, time = Time.now) end
```

Reply:
![スクリーンショット 2020-05-16 2 22 54](https://user-images.githubusercontent.com/4254002/82078532-360e3280-971c-11ea-8aab-0abf64cf2341.png)
手元のyardでは実行できて、期待した表示になっていました。

```
➤ bundle exec yard --version
yard 0.9.24
➤ bundle exec rake yard
(in /Users/ysakasin/src/BCDice)
[warn]: Unknown tag @params in file `src/bcdiceCore.rb` near line 994
[warn]: Unknown tag @params in file `src/bcdiceCore.rb` near line 1015
[warn]: Unknown tag @params in file `src/bcdiceCore.rb` near line 1056
[warn]: Unknown tag @params in file `src/bcdiceCore.rb` near line 1056
[warn]: Unknown tag @params in file `src/bcdiceCore.rb` near line 1056
[warn]: Unknown tag @tableData in file `src/TableFileData.rb` near line 30
[warn]: Unknown tag @abstruct in file `src/diceBot/DiceBot.rb` near line 339
[warn]: Unknown tag @abstruct in file `src/diceBot/DiceBot.rb` near line 344
[warn]: Unknown tag @abstruct in file `src/diceBot/DiceBot.rb` near line 349
[warn]: Unknown tag @abstruct in file `src/diceBot/DiceBot.rb` near line 354
[warn]: Unknown tag @abstruct in file `src/diceBot/DiceBot.rb` near line 359
[warn]: Unknown tag @sendMode
[warn]: Unknown tag @sortType
[warn]: Unknown tag @sameDiceRerollCount
[warn]: Unknown tag @sameDiceRerollType
[warn]: Unknown tag @d66Type
[warn]: Unknown tag @isPrintMaxDice
[warn]: Unknown tag @upplerRollThreshold
[warn]: Unknown tag @unlimitedRollDiceType
[warn]: Unknown tag @rerollNumber
[warn]: Unknown tag @defaultSuccessTarget
[warn]: Unknown tag @rerollLimitCount
[warn]: Unknown tag @fractionType
[warn]: Unknown tag @fractionType
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 84
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 102
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 120
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 138
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 156
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 174
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 192
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 210
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 228
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 246
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 264
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 282
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 300
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 318
[warn]: Unknown tag @override in file `src/diceBot/MeikyuKingdomBasic.rb` near line 336
[warn]: @param tag has unknown parameter name: bot 
    in file `src/test/test_dicebot_info_is_defined.rb' near line 26
[warn]: @param tag has unknown parameter name: bot 
    in file `src/test/test_dicebot_info_is_defined.rb' near line 35
[warn]: @param tag has unknown parameter name: bot 
    in file `src/test/test_dicebot_info_is_defined.rb' near line 44
[warn]: @param tag has unknown parameter name: bot 
    in file `src/test/test_dicebot_info_is_defined.rb' near line 53
Files:         222
Modules:         4 (    1 undocumented)
Classes:       244 (  200 undocumented)
Constants:     807 (   70 undocumented)
Attributes:     62 (    0 undocumented)
Methods:      2506 ( 1610 undocumented)
 48.08% documented
```

Reply:
すみません、改めて試したらきちんと表示されていました🙇　手元で編集してしまっていたかもしれません。